src/styles/createMuiTheme.d.ts

```diff
@@ -6,12 +6,15 @@ import { Spacing } from './spacing';
 import { Transitions } from './transitions';
 import { Typography, TypographyOptions } from './createTypography';
 import { ZIndex } from './zIndex';
+import { StyleRules } from './withStyles'
 
 export interface ThemeOptions {
   breakpoints?: Partial<BreakpointsOptions> & Partial<Breakpoints>;
   mixins?: Partial<Mixins>;
   palette?: Partial<Palette>;
   typography?: TypographyOptions | ((palette: Palette) => TypographyOptions);
+  shadows?: Shadows;
+  [propName: string]: Object;
```

Comment:
Instead of just a blanket `[propName: string]: Object` it would be better to enumerate the other missing theme properties.

Reply:
Thanks, ok. It will respect the code from styles\createMuiTheme.js, especially
```
    ...deepmerge(
      {
        transitions,
        spacing,
        zIndex,
      },
      other,
    ),
```
See new version please.